src/test/test_pivx.cpp

```diff
@@ -139,6 +139,8 @@ TestChainSetup::TestChainSetup(int blockCount) : TestingSetup(CBaseChainParams::
         CBlock b = CreateAndProcessBlock(noTxns, scriptPubKey);
         coinbaseTxns.push_back(*b.vtx[0]);
     }
+
+    deterministicMNManager->UpdatedBlockTip(chainActive.Tip());
```

Comment:
is this forced update needed?
The validationinterface should be signaling the `UpdateBlockTip` event on every block connection.

Reply:
Mhm...
At the moment this is indeed needed, because we have no EvoNotificationInterface registered for the tests.
But yeah, this comment makes me realize, it would be much better to introduce and register the interface in the TestSetup fixture, instead, and remove all direct calls to `UpdatedBlockTip` :+1:
Have added a commit for this at the end.

Reply:
good 👍